static/js/apps/nl_interface/app.tsx

```diff
@@ -34,6 +34,9 @@ export function App(): JSX.Element {
   const [contextList, setContextList] = useState<any[]>([]);
   const urlPrompts = useRef(getUrlPrompts());
   const autoRun = useRef(getUrlToken("a"));
```

Comment:
curious why autorun needs to be a ref? the value shouldn't change once the page has loaded (or rather, we won't need to listen to changes..)

Reply:
oh yeah, this doesn't need to be a ref anymore, removed the ref part!